cluster/local/integration_tests.sh

```diff
@@ -206,7 +206,7 @@ wait_for_pods_in_namespace 120 "kube-system" "${kindpods[@]}"
 # install crossplane from master channel
 echo_step "installing crossplane from master channel"
 "${HELM}" repo add crossplane-master https://charts.crossplane.io/master/
-chart_version="$("${HELM}" search crossplane-master/crossplane | awk 'FNR == 2 {print $2}')"
+chart_version="$("${HELM}" search crossplane-master/crossplane --devel | awk 'FNR == 2 {print $2}')"
```

Comment:
Note I changed this as we were actually essentially using the `alpha` channel for testing because whenever helm was upgraded to require the `--devel` flag it was not included. This should now actually install from `master` channel, which is a different test, but I think it is what we were trying to achieve originally.